x-pack/plugins/apm/public/components/shared/charts/transaction_distribution_chart/index.tsx

```diff
@@ -137,15 +131,15 @@ export function TransactionDistributionChart({
 
   // This will create y axis ticks for 1, 10, 100, 1000 ...
   const yMax =
-    Math.max(...(overallHistogram ?? []).map((d) => d.doc_count)) ?? 0;
+    Math.max(
+      ...flatten(data.map((d) => d.histogram)).map((d) => d.doc_count)
```

Comment:
Can you use flatMap here?
```suggestion
      ...data.flatMap((d) => d.histogram).map((d) => d.doc_count)
```